torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -824,9 +809,9 @@ std::vector<torch::lazy::BackendDataPtr> XLAGraphExecutor::ExecuteStablehlo(
       runtime::GetComputationClient()->ExecuteComputation(
           *computations[0], UnwrapXlaData(arguments), device.toString());
 
+  std::vector<torch::lazy::BackendDataPtr> placeholders;
   for (size_t i = 0; i < results.size(); ++i) {
-    XLA_CHECK(placeholders[i] != nullptr);
-    placeholders[i]->Assign(*WrapXlaData(results[i]));
+    placeholders.push_back(WrapXlaData(results[i]));
```

Comment:
nit you should not call them placeholders, you can just call it `results` or `datas`